pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -117,9 +118,16 @@ func removeDaemonTask() error {
 }
 
 func checkIfDaemonTaskRunning() error {
-	if err := olderDaemonVersionRunning(); err != nil {
-		return err
+	// First check if a daemon process is running and then check if
+	// the running daemon process is of older version, this way  we
+	// return nil if a daemon process is running that is not old
+	if _, err := daemonclient.GetVersionFromDaemonAPI(); err != nil {
```

Comment:
@anjannath if daemonAPI is not running then you will get the err I think condition should be 
```
	if _, err := daemonclient.GetVersionFromDaemonAPI(); err == nil {
		if err := olderDaemonVersionRunning(); err != nil {
			return err
		}
		return nil
	}

```

Reply:
yes, thanks, i was pulling my hair why it wasn't working as expected.

Reply:
@praveenkumar Updated

Reply:
Fwiw, this could be refactored some more if this code stays longer term, `olderDaemonVersionRunning()` is just `GetVersionFromDaemonAPI` + `CheckIfOlderVersion`, so it's suboptimal to wrap it in a call to `GetVersionFromDaemonAPI`. But good enough for now.

Hopefully this won't stay long term anyway, and will only be a temporary fix to get the release out, as I said before, I think we should only support:
- daemon configured to autostart with `crc setup`
- daemon started manually after setting the appropriate `skip-xxx` config options.

Right now, the daemon is not autostarted on Windows, so we have a gap to fill. Checking if the daemon is already running is what was done before, so we can readd this as a stop gap, especially as this is in Windows-specific code.